lib/navigation-bar/index.tsx

```diff
@@ -127,22 +143,21 @@ export class NavigationBar extends Component {
   }
 }
 
-const mapStateToProps = ({ appState: state, settings }) => ({
+const mapStateToProps = ({ appState: state, ui, settings }) => ({
   autoHideMenuBar: settings.autoHideMenuBar,
   dialogs: state.dialogs,
   selectedTag: state.tag,
-  showNavigation: state.showNavigation,
+  showNavigation: ui.showNavigation,
   showTrash: state.showTrash,
 });
 
 const {
   showAllNotesAndSelectFirst,
   selectTrash,
   showDialog,
-  toggleNavigation,
 } = appState.actionCreators;
 
-const mapDispatchToProps = {
+const mapDispatchToProps: S.MapDispatch<DispatchProps, OwnProps> = {
```

Comment:
note that when we don't use or rely on `ownProps` then we don't have any need to include it in the type signature. in fact, _any_ time we use the object form of `mapDispatchToProps` (vs. the function form `(dispatch, ownProps) => …`) then we don't need to include the `OwnProps` in the signature.

on a deeper level, if you inspect the type you can see that it uses a default parameter much the same way a normal functions does…

```ts
type MapDispatch<DispatchProps, OwnProps = {}> = { … }
```